src/pool.jl

```diff
@@ -189,7 +180,8 @@ function merge_pools(a::CategoricalPool{T}, b::CategoricalPool) where {T}
         newlevs = copy(levels(a))
         ordered = isordered(a)
```

Comment:
This line and also line 178 and 175 are inconsistent with the description you gave. Namely you say that pool is marked unordered if any of merged pools is unordered. But these two lines follow a different rule if one of them is empty (as then it inherits orderedness from `a` or `b`)